frontend/my-adopthive-app/src/AdopteesProfile/AdopteesProfile.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect, useState } from 'react';
+import './AdopteesProfile.css'
+
+const AdopteesProfile = () => {
+  const [Adoptee, setAdoptee] = useState([]);
+
+  useEffect(() => {
+    const fetchAdoptee = async () => {
+      try {
+        const response = await fetch('http://localhost:3001/adoptee-profile'); // Replace with your backend URL
+        if (!response.ok) {
+          throw new Error('Network response was not ok');
+        }
+        const data = await response.json();
+        setAdoptee(data);
+      } catch (error) {
+
```

Comment:
Mentioned this before but let's have some more graceful error handling rather than an empty catch statement

Reply:
Can I do return(error) after the catch(error)